db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+10.times do |i|
+  Book.create(name:"book#{i}",price:(i+1)*100,author_id:1)
```

Comment:
Instead of author_id 1 add reference of author by using Author.first or Author.find_by(first_name: "auth#{i}")